assets/javascripts/kitten/components/tooltips/tooltip.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import TooltipIcon from './tooltip-icon'
+import ReactTooltip from 'react-tooltip'
+
+export default class Tooltip extends React.Component {
+  render() {
+    return (
+      <div>
+        <TooltipIcon data-tip
+                     data-for="tooltip"
+                     data-event="click" />
```

Comment:
Ca peut être intéressant d'avoir une option de positionnement avec `place={ this.props.place }`.